src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -106,6 +106,9 @@
       <_Dependency Remove="Newtonsoft.Json"/>
       <_Dependency Remove="stdole"/>
       <_Dependency Remove="StreamJsonRpc"/>
+      <_Dependency Remove="Nerdbank.Streams"/>
+      <_Dependency Remove="System.IO.Pipelines"/>
+      <_Dependency Remove="System.ValueTuple"/>
```

Comment:
Add a comment that this is only needed until StreamJsonRpc includes `net472` target and file an issue to update.

Reply:
okay